internal/fetch/fetch.go

```diff
@@ -0,0 +1,60 @@
+package fetch
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/docker/app/internal/relocated"
+	"github.com/docker/cnab-to-oci/relocation"
+	"github.com/docker/go/canonical/json"
+	"github.com/pkg/errors"
+)
+
+func BundleJSON(bundlePath string) (*bundle.Bundle, error) {
+	data, err := ioutil.ReadFile(bundlePath)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to read file %s", bundlePath)
+	}
+	bndl, err := bundle.Unmarshal(data)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to read file %s", bundlePath)
+	}
+	return bndl, nil
+}
+
+func RelocationMapJSON(relocationMapPath string) (relocation.ImageRelocationMap, error) {
```

Comment:
nit: it looks like this function is only used here in this package, so it may not be exported.